LongWarOfTheChosen/Localization/XComGame.fra

```diff
@@ -38,7 +38,7 @@ m_strLowEngineersList[2]="Un ingénieur est parfois disponible au marché noir e
 m_strRetributionAlertTitle="VENGEANCE DES ÉLUS"
 m_strRetributionAlertDescription="Un Élu limite les activités de la résistance dans cette région, ce qui réduit l'efficacité des actions entreprises dans ce secteur." 
 m_strRetributionAlertIncomeDecreasedTitle="REFUGE SABOTÉ"
-m_strRetributionAlertIncomeDescription="L'efficacité du refuge lors des missions de Ressources, Renseignements et Recruter est réduite de 20%."
+m_strRetributionAlertIncomeDescription="L'efficacité du refuge lors des missions de Ressources, Renseignements et Recruter est réduite de 20 %."
```

Comment:
Is the extra space here intentional?

Reply:
Yes. All the lines with a % in it have a space behind the char.